src/modules/Client/Client.tsx

```
@@ -0,0 +1,484 @@
+  // Filter clients when searchTerm changes
+  useEffect(() => {
+    if (searchTerm.trim() === '') {
+      setFilteredClients(clients);
+    } else {
+      const lowerCaseSearch = searchTerm.toLowerCase();
+      const filtered = clients.filter(
+        (client) =>
+          client.firstName.toLowerCase().includes(lowerCaseSearch) ||
+          client.lastName.toLowerCase().includes(lowerCaseSearch) ||
+          client.email.toLowerCase().includes(lowerCaseSearch) ||
+          (client.phoneNumber && client.phoneNumber.includes(searchTerm))
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)